package.json

```diff
@@ -1,43 +1,46 @@
 {
-  "name": "tutorial",
+  "name": "recipe-app",
   "version": "1.0.0",
   "scripts": {
-    "devStart": "nodemon server.js",
+    "devStart": "nodemon --config nodemon.json ./src/server.ts",
     "start": "ts-node ./src/server.ts"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "bcrypt": "^5.0.1",
-    "cookie-parser": "^1.4.5",
-    "express": "^4.17.1",
-    "express-generator-typescript": "^1.7.8",
-    "jsonwebtoken": "^8.5.1",
-    "mongoose": "^5.12.11"
+    "bcrypt": "5.0.1",
+    "cookie-parser": "1.4.5",
+    "express": "4.17.1",
+    "jsonwebtoken": "8.5.1",
+    "mongoose": "5.12.11"
   },
   "nodemonConfig": {
+    "restartable": "rs",
     "watch": [
       "src"
     ],
-    "ext": "ts, html",
+    "ext": "ts, html, json",
     "ignore": [
-      "src/public"
+      "src/public",
+      ".git",
+      "node_modules/"
     ],
     "exec": "./node_modules/.bin/ts-node -r tsconfig-paths/register ./src"
   },
   "devDependencies": {
-    "@types/bcrypt": "^5.0.0",
-    "@types/cookie-parser": "^1.4.2",
-    "@types/express": "^4.17.12",
-    "@types/jsonwebtoken": "^8.5.3",
-    "@types/node": "^16.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.28.2",
-    "@typescript-eslint/parser": "^4.28.2",
-    "dotenv": "^10.0.0",
-    "nodemon": "^2.0.9",
-    "ts-node": "^10.0.0",
-    "typescript": "^4.3.5"
+    "@types/bcrypt": "5.0.0",
+    "@types/cookie-parser": "1.4.2",
+    "@types/express": "4.17.12",
+    "@types/jsonwebtoken": "8.5.3",
+    "@types/mongoose": "5.11.97",
+    "@types/node": "16.0.0",
+    "@typescript-eslint/eslint-plugin": "4.28.2",
+    "@typescript-eslint/parser": "4.28.2",
+    "dotenv": "10.0.0",
```

Comment:
dotenv should be in dependencies, not devDependencies

Reply:
idk how it happened